package.json

```diff
@@ -8,8 +8,9 @@
     "react-dom": ">=0.14.0 || ^0.14.0-beta3"
   },
   "dependencies": {
-    "create-react-class": "^15.5.2",
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
```

Comment:
This shouldn't be a dependency.  It would remove our ability to support old versions.
React-tooltip should also not be a dependency.  Please revert these.

Why did you need to change the prop-types support?  If there's no good reason we should revert that as well.

Reply:
ok then I will move this back to devDependencies and revert prop-types version